<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QMainWindow" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班管理系统 - 管理员</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="flightManageTab">
       <attribute name="title">
        <string>航班管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="flightLayout">
        <item>
         <layout class="QHBoxLayout" name="flightToolbar">
          <item>
           <widget class="QPushButton" name="addFlightBtn">
            <property name="text">
             <string>添加航班</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editFlightBtn">
            <property name="text">
             <string>编辑航班</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteFlightBtn">
            <property name="text">
             <string>删除航班</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshFlightBtn">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="flightSearchEdit">
            <property name="placeholderText">
             <string>搜索航班号...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchFlightBtn">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="flightTable">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userManageTab">
       <attribute name="title">
        <string>用户管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="userLayout">
        <item>
         <layout class="QHBoxLayout" name="userToolbar">
          <item>
           <widget class="QPushButton" name="viewUserBtn">
            <property name="text">
             <string>查看详情</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetPasswordBtn">
            <property name="text">
             <string>重置密码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshUserBtn">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="userSearchEdit">
            <property name="placeholderText">
             <string>搜索用户名...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchUserBtn">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="userTable">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statisticsTab">
       <attribute name="title">
        <string>统计报表</string>
       </attribute>
       <layout class="QVBoxLayout" name="statsLayout">
        <item>
         <layout class="QHBoxLayout" name="statsToolbar">
          <item>
           <widget class="QComboBox" name="statsTypeCombo">
            <item>
             <property name="text">
              <string>总体统计</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>航班统计</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>用户统计</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>收入统计</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="statsFromDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="toLabel">
            <property name="text">
             <string>至</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="statsToDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="generateStatsBtn">
            <property name="text">
             <string>生成报表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportStatsBtn">
            <property name="text">
             <string>导出报表</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="statsTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="systemTab">
       <attribute name="title">
        <string>系统管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="systemLayout">
        <item>
         <widget class="QGroupBox" name="dbGroupBox">
          <property name="title">
           <string>数据库管理</string>
          </property>
          <layout class="QHBoxLayout" name="dbLayout">
           <item>
            <widget class="QPushButton" name="backupDbBtn">
             <property name="text">
              <string>备份数据库</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="restoreDbBtn">
             <property name="text">
              <string>恢复数据库</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="optimizeDbBtn">
             <property name="text">
              <string>优化数据库</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="logGroupBox">
          <property name="title">
           <string>系统日志</string>
          </property>
          <layout class="QVBoxLayout" name="logLayout">
           <item>
            <layout class="QHBoxLayout" name="logToolbar">
             <item>
              <widget class="QPushButton" name="refreshLogBtn">
               <property name="text">
                <string>刷新日志</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearLogBtn">
               <property name="text">
                <string>清空日志</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="exportLogBtn">
               <property name="text">
                <string>导出日志</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="logTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionImportData"/>
    <addaction name="actionExportData"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionChangePassword"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUserManual"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportData">
   <property name="text">
    <string>导入数据...</string>
   </property>
  </action>
  <action name="actionExportData">
   <property name="text">
    <string>导出数据...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>偏好设置...</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>修改密码...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
  <action name="actionUserManual">
   <property name="text">
    <string>使用手册</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui> 